README.md

```diff
@@ -0,0 +1,5 @@
+# Atoms on Flathub
+
+## Integrations
+### Distrobox
+To enable Distrobox integration add `--talk-name=org.freedesktop.Flatpak` and restart the application.
```

Comment:
Just note that this is unlikely to be ever found by a regular user. It would be better to have it documented on the project website and/or the app itself.

Reply:
I am going to create a documentation of the project and move it there. Thanks for your advice.